src/main/kotlin/com/chattriggers/ctjs/minecraft/wrappers/Player.kt

```diff
@@ -195,6 +195,9 @@ object Player {
     @JvmStatic
     fun getLightLevel(): Int = World.getWorld()?.getLight(getPlayer()?.position) ?: 0
 
+    @JvmStatic
+    fun isMoving(): Boolean = getPlayer()?.let { it.movementInput.moveForward != 0F || it.movementInput.moveStrafe != 0F } ?: false
```

Comment:
Small nitpick to shorten the line a bit:
```suggestion
    fun isMoving(): Boolean = getPlayer()?.movementInput?.let { it.moveForward != 0F || it.moveStrafe != 0F } ?: false
```